systemtest/src/test/java/io/strimzi/systemtest/operators/topic/TopicReplicasChangeST.java

```diff
@@ -0,0 +1,497 @@
+
+        LOGGER.info("ReplicaChange appeared for KafkaTopic: {}/{}. Proceeding to the next stage.", this.sharedTestStorage.getNamespaceName(), testStorage.getTopicName());
+
+        // we should see that KafkaTopic is ongoing
+        KafkaTopicUtils.waitUntilReplicaChangeOngoing(this.sharedTestStorage.getNamespaceName(), testStorage.getTopicName());
+
+        final Map<String, String> ccPod = DeploymentUtils.depSnapshot(this.sharedTestStorage.getNamespaceName(), CruiseControlResources.componentName(this.sharedTestStorage.getClusterName()));
+
+        // --- 3rd stage (during ongoing replicaChange task we induce chaos and delete CruiseControl Pod)
+        kubeClient().deletePodWithName(this.sharedTestStorage.getNamespaceName(), ccPodName);
+
+        // we should see that KafkaTopic is ongoing
+        KafkaTopicUtils.waitUntilReplicaChangeOngoing(this.sharedTestStorage.getNamespaceName(), testStorage.getTopicName());
+
+        // then KafkaTopic replicaChange status should disappear after task is completed by CC
+        KafkaTopicUtils.waitForReplicaChangeStatusNotPresent(this.sharedTestStorage.getNamespaceName(), testStorage.getTopicName());
+
+        // cc should roll
+        DeploymentUtils.waitTillDepHasRolled(this.sharedTestStorage.getNamespaceName(), CruiseControlResources.componentName(this.sharedTestStorage.getClusterName()), 1, ccPod);
+
+        verifyKafkaTopicAfterReplicationChange(testStorage, topicObservationGeneration, increasedTopicReplicationFactor);
+
+        // exchange messages within such Topic to prove that its operative
+        sendAndRecvMessages(testStorage);
+    }
+
+    /**
+     * @description Evaluates the system's ability to recover from a replication factor change operation during an unexpected
+     * Entity Operator crash, ensuring that KafkaTopic changes are successfully applied and the system remains operational.
+     *
+     * @steps
+     *  1. - Initiate a replication factor change for a KafkaTopic.
+     *     - Replication change is in progress.
+     *  2. - Simulate an Entity Operator crash during the replication factor change process.
+     *     - Entity Operator crashes but is subsequently recovered.
+     *  3. - Confirm that the replication factor change is successfully completed after the recovery.
+     *     - The replication factor change is completed successfully, demonstrating system robustness.
+     *
+     * @usecase
+     *  - Robustness in handling component failures
+     *  - Successful application of KafkaTopic changes post-recovery
+     */
+    @IsolatedTest
+    void testRecoveryOfReplicationChangeDuringEoCrash() {
+        final TestStorage testStorage = new TestStorage(ResourceManager.getTestContext());
+        final int topicPartitions = 3;
+        final int startingTopicReplicationFactor = 3;
+        final Pod eoPod = kubeClient().listPods(this.sharedTestStorage.getNamespaceName(),
```

Comment:
You don't need to get the whole `Pod` object, just the name of the Pod or?
Also, do you need to use `this` in every test? 
(just a small nits).

Reply:
> You don't need to get the whole Pod object, just the name of the Pod or?

true, thanks

> Also, do you need to use this in every test? (just a small nits).

well...that's just a matter of style... :) I think maybe we should be consistent with respect to other classes so I can remove `this`

Reply:
I have removed `this.`